ui/tests/acceptance/secrets/backend/ssh/configuration-test.js

```diff
@@ -90,9 +86,9 @@ module('Acceptance | ssh | configuration', function (hooks) {
     );
     assert.dom(GENERAL.maskedInput('privateKey')).hasNoText('Private key is empty and reset');
     assert.dom(GENERAL.inputByAttr('publicKey')).hasNoText('Public key is empty and reset');
-    assert
-      .dom(GENERAL.inputByAttr('generate-signing-key-checkbox'))
-      .isNotChecked('Generate signing key is unchecked');
+    // change in behavior after refactor. Because we refresh the model after delete, the generate signing key is checked by default.
```

Comment:
Thanks for this comment! IMO it doesn't need to live in the code (could be a github comment) but that's a very small nit 😄